lib/osf-components/addon/components/registries/finalize-registration-modal/component.ts

```diff
@@ -44,13 +44,18 @@ export default class FinalizeRegisrationModalComponent extends Component {
     @computed('manager.draftManager.reviewsWorkflow')
     get noticeText() {
         const translationOptions = { learnMoreLink: this.learnMoreLink, htmlSafe: true };
-        if (this.manager.draftManager.reviewsWorkflow) {
-            return this.intl.t(
-                'registries.finalizeRegistrationModal.notice.withModerationFromProject',
-                translationOptions,
-            );
+        let translationString = '';
+
+        if (this.manager.draftManager.node) {
+            translationString = this.manager.draftManager.reviewsWorkflow
+                ? 'withModerationFromProject'
+                : 'noModerationFromProject';
+        } else {
+            translationString = this.manager.draftManager.reviewsWorkflow
+                ? 'withModerationNoProject'
+                : 'noModerationNoProject';
```

Comment:
@baylee-d I think what you want to use here is `draftManager.draftRegistration.hasProject` instead of checking `draftManager.node`. `draftManager.node` corresponds to `draftRegistration.branchedFrom` which will be a `DraftNode` in the case of a no-project draft?